app/models/pseud.rb

```diff
@@ -5,7 +5,13 @@ class Pseud < ApplicationRecord
 
   has_attached_file :icon,
     styles: { standard: "100x100>" },
-    path: %w(staging production).include?(Rails.env) ? ":attachment/:id/:style.:extension" : ":rails_root/public:url",
+    path: if Rails.env.production?
+            ":attachment/:id/:style.:extension"
+          elsif Rails.env.staging?
+            "staging/:attachment/:id/:style.:extension"
+          else
+            ":rails_root/public:url"
```

Comment:
All the uploaded files end up in the public folder. So I think we also need to:

- Change the path to include the environment, so we can clean up test data without touching development data. `url` is `/system/:class/:attachment/:id_partition/:style/:filename`, so we can use `:rails_root/public/system/:rails_env/:class/:attachment/:id_partition/:style/:filename`.
- Add `public/system/test` and `public/system/development` to .gitignore.
- Add [an after suite hook to `FileUtils.rm_rf` the test folder](https://github.com/thoughtbot/paperclip/#testing).